plugin/src/main/java/com/playmonumenta/scriptedquests/managers/ZonePropertyManager.java

```diff
@@ -81,4 +79,8 @@ private void changeEvent(Player player, String layerName, String name) {
 
 		entry.changeEvent(mPlugin, player);
 	}
+
+	public Map<String, Map<String, ZoneProperty>> getZoneProperties() {
+		return mZoneProperties;
```

Comment:
This should create a new map of maps so the original is "immutable" (unless there's a better way to pull that off)

Reply:
Collections.UnmodifiableMap() maybe?

Reply:
Added that